intake_aodn/indexing.py

```diff
@@ -119,7 +119,8 @@ def process_aggregate(root='imos-data/IMOS/SRS/SST/ghrsst/L3S-1d/ngt/',
                     refs,
                     remote_protocol="s3",
                     remote_options=storage_options,
-                    xarray_concat_args=dict(dim='time',coords='minimal',join='override',compat='override',combine_attrs='override', fill_value=''),
+                    concat_dims=["time"], coo_map={"time": "data:time"},
```

Comment:
Heh so simple, just read the docs about this too. Haven't looked at the code but it is probably quite a bit quicker than xarray concat. I would just remove the old code rather than just comment out.

Reply:
I agree there were a few other things in the notebook but quite straightforward

Reply:
I've updated the code and also put the keep fields function in the indexing.py